src/osd/PG.cc

```diff
@@ -3789,6 +3789,21 @@ void PG::handle_query_state(Formatter *f)
     f->dump_stream("scrubber.max_end") << scrubber.max_end;
     f->dump_stream("scrubber.subset_last_update") << scrubber.subset_last_update;
     f->dump_bool("scrubber.deep", scrubber.deep);
+    f->dump_bool("scrubber.must_scrub", scrubber.must_scrub);
```

Comment:
Move all this in scrubber new dump() and only output fields that aren't reset() if not in an active scrub.